lib/productions/extended-attributes.js

```diff
@@ -1,8 +1,41 @@
 import { Base } from "./base.js";
 import { ArrayBase } from "./array-base.js";
-import { list, identifiersOrStrings, argument_list, autoParenter } from "./helpers.js";
+import { Token } from "./token.js";
+import { list, argument_list, autoParenter } from "./helpers.js";
 import { validationError } from "../error.js";
 
+/**
+ * @param {import("../tokeniser").Tokeniser} tokeniser
+ */
+function identifiers(tokeniser) {
```

Comment:
Oops, forgot that we are replacing `identifiers` with `identifiersOrStrings`. That means it's safe to remove this!